src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -348,10 +348,10 @@ export class UiSettingsClient implements IUiSettingsClient {
         this.log.warn(
           `Deprecation warning: The setting "${key}" has multiple scopes. Please specify a scope when updating it.`
         );
-      } else if (this.userLevelSettingsKeys.includes(key)) {
-        groupedChanges[UiSettingScope.USER][key] = val;
       } else if (this.adminUiSettingsKeys.includes(key)) {
         groupedChanges[UiSettingScope.DASHBOARD_ADMIN][key] = val;
+      } else if (this.userLevelSettingsKeys.includes(key)) {
```

Comment:
Does the order matter? If yes, would be nice to add a comment or unit test

Reply:
This is to maintain the existing logic hierarchy that the admin-level configurations should has higher privilege level as initiatively admin setting > user settings > workspace settings > global settings.